docs/reference/modules/settings.md

```diff
@@ -0,0 +1,127 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/settings`
+
+**Alias:** `apos.settings`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module governs the Personal Settings menu that allows users to change their preferences. Apostrophe projects can configure this menu through `@apostrophecms/settings`.
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`subforms`](#subforms) | Object | Each object defines an input field in the Personal Settings menu. |
```

Comment:
a set of input fields in the Personal Settings dialog box, with its own Update button.